src/util/dump.ts

```diff
@@ -147,4 +148,18 @@ export async function importPolyfillModules(zimCreator: ZimCreator) {
         });
         zimCreator.addArticle(article);
     });
-}
\ No newline at end of file
+
+    const webpHandlerUrl = 'https://gist.githubusercontent.com/rgaudin/60bb9cc6f187add506584258028b8ee1/raw/9d575b8e25d67eed2a9c9a91d3e053a0062d2fc7/web-handler.js';
+    const content = await axios.get(webpHandlerUrl, {headers: {}, responseType: 'arraybuffer', timeout: 60000, method: 'GET', validateStatus(status) { return (status >= 200 && status < 300) || status === 304; }})
```

Comment:
is `method: 'GET'` needed here? we are already using `axios.get()` here

Reply:
Right ; simplified